packages/wallets/src/components/AddressBookAutocomplete.tsx

```diff
@@ -0,0 +1,81 @@
+import type { AddressContact } from '@chia-network/core';
+import { AddressBookContext } from '@chia-network/core';
+import { Trans } from '@lingui/macro';
+import { Autocomplete as MaterialAutocomplete, FormControl, TextField, TextFieldProps } from '@mui/material';
+import React, { useEffect, useState, useContext } from 'react';
+import { useController, useFormContext } from 'react-hook-form';
+
+type Props = TextFieldProps &
+  AddressBookAutocompleteProps<string, false, false, true> & {
+    name: string;
+    getType: string; // currently supports 'address' and 'did'
+    fullWidth?: boolean;
+    freeSolo?: boolean;
+    renderInput?: any;
+  };
+
+export default function AddressBookAutocomplete(props: Props) {
+  const { name, getType, required, fullWidth, freeSolo, disableClearable, ...rest } = props;
+  const [addressBook, ,] = useContext(AddressBookContext);
+  const [options, setOptions] = useState([]);
+  const { control } = useFormContext();
+
+  const {
+    field: { onChange, onBlur },
+  } = useController({
+    name,
+    control,
```

Comment:
you can remove control, it will be used automatically

Reply:
Tracked here: https://github.com/Chia-Network/issue-tracker/issues/537